From: Phil Elwell <phil@raspberrypi.com>
Date: Sat, 4 Jul 2020 22:19:26 +0100
Subject: overlays: rpi-ft5406: Fix boolean parameters

An improvement in the automated testing of overlays revealed
these invalid boolean parameter declarations.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts b/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
index 4dcb490f04b6..8483c4f4b2eb 100644
--- a/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
@@ -18,8 +18,8 @@
 	__overrides__ {
 		touchscreen-size-x = <&ts>,"touchscreen-size-x:0";
 		touchscreen-size-y = <&ts>,"touchscreen-size-y:0";
-		touchscreen-inverted-x = <&ts>,"touchscreen-inverted-x:?";
-		touchscreen-inverted-y = <&ts>,"touchscreen-inverted-y:?";
-		touchscreen-swapped-x-y = <&ts>,"touchscreen-swapped-x-y:?";
+		touchscreen-inverted-x = <&ts>,"touchscreen-inverted-x?";
+		touchscreen-inverted-y = <&ts>,"touchscreen-inverted-y?";
+		touchscreen-swapped-x-y = <&ts>,"touchscreen-swapped-x-y?";
         };
 };
